lib/consul-mesh-extension.ts

```diff
@@ -27,6 +27,13 @@ export enum CloudProviders {
     HASHICORP_CLOUDPROVIDER = 'hcp'
 }
 
+export interface ContainerMutatingProps {
+    /**
+     * Application Container Health Checks
+     */
```

Comment:
nit: @default doc for this prop (basically all optional properties of an interface should have one)